packages/angular_devkit/build_angular/test/protractor/works_spec_large.ts

```diff
@@ -5,47 +5,58 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+import { Architect } from '@angular-devkit/architect/src/index2';
+import { JsonObject, normalize } from '@angular-devkit/core';
+import { createArchitect, host, protractorTargetSpec } from '../utils';
 
-import { runTargetSpec } from '@angular-devkit/architect/testing';
-import { normalize } from '@angular-devkit/core';
-import { retry } from 'rxjs/operators';
-import { host, protractorTargetSpec } from '../utils';
+describe('Protractor Builder', () => {
```

Comment:
Any reason to remove the `retry()` on some of them?  Is it more stable now?

Reply:
Appears more stable.  I've been running them repeatedly over the last week without issue.  I think it also makes the tests more representative as well.  Ideally, we shouldn't have to repeat tests to make them pass.